core/src/main/resources/jenkins/model/Jenkins/login.jelly

```diff
@@ -83,6 +83,7 @@ THE SOFTWARE.
                 </j:if>
                 <div class="formRow">
                   <input
+                    aria-label="Username"
```

Comment:
```suggestion
                    aria-label="${%Username}"
```

This makes the `aria-label` localizable, and actually reuses translations for the placeholder.

Not sure whether the latter is a good idea though.